benchmark/scripts/Benchmark_Driver

```diff
@@ -205,24 +212,101 @@ class BenchmarkDoctor(object):
                 'See http://bit.ly/UpperCamelCase')
 
     @staticmethod
-    def _name_is_at_most_40_chars_long(test):
-        if len(test) > 40:
-            BenchmarkDoctor.log_naming.warning(
-                "'%s' name is longer than 40 characters.", test)
+    def _name_is_at_most_40_chars_long(measurements):
```

Comment:
Not sure if this rule makes sense. If longer names cause troubles in reports, we should rather fix the reports

Reply:
I have tried my best to fix the reports before, see the still open #9999. But there is no way to make it work on mobile. GitHub doesn’t give us much control over the CSS, which is the key.

If we follow your previous *rename-benchmark-when-runtime-changes-significantly* rule, we will be renaming most (all?) of the tests that violate the *max-length-40* naming rule while fixing the *runtime-under-2500-microseconds* violations anyway.